### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by artempolischuk4315**

### CODE REVIEW

File: src/main/java/polishchuk/controller/PlayerController.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    private PlayerService playerService;
+
+    @Autowired
+    public PlayerController(PlayerService playerService) {
+        this.playerService = playerService;
+    }
+
+
+    @GetMapping("/players/{id}")
```
Comment (valeriiafatova) [inline src/main/java/polishchuk/controller/PlayerController.java]: Move common path to Controller mapping

File: src/main/java/polishchuk/controller/TeamController.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        this.teamService = teamService;
+    }
+
+
+    @GetMapping("/teams/{id}")
+    public TeamDto getTeam(@PathVariable Integer id){
+        return teamService.findById(id);
+    }
+
+    @PostMapping("/teams/creator")
```
Comment (valeriiafatova) [inline src/main/java/polishchuk/controller/TeamController.java]: Very bad. No verbs, we use only resources and http methods

File: src/main/java/polishchuk/controller/TeamController.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    public TeamDto createTeam(@RequestBody TeamDto team){
+        return teamService.save(team);
+    }
+
+
+    @PutMapping("/teams/updater/{playerId}")
```
Comment (valeriiafatova) [inline src/main/java/polishchuk/controller/TeamController.java]: Same.

File: src/main/java/polishchuk/controller/TeamController.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    public Map<String, Integer> updateTeam
```
Comment (valeriiafatova) [inline src/main/java/polishchuk/controller/TeamController.java]: What is the return type? Architect mistake

Comment (artempolischuk4315) [reply src/main/java/polishchuk/controller/TeamController.java]: Finished fixing

File: src/main/java/polishchuk/controller/PlayerController.java (hunk 1 above)
Comment (artempolischuk4315) [reply src/main/java/polishchuk/controller/PlayerController.java]: Done

**Pull request merged**